06-TimeManagement/user_space/log.txt

```diff
@@ -0,0 +1,79 @@
+$ ./myprog 
+User-space get absolute time: 1573486483.379240
+Kernel-space get absolute time: 1573486483.378988
+
+$ ./myprog 
+User-space get absolute time: 1573486487.784704
+Kernel-space get absolute time: 1573486487.784672
+
+$ ./myprog 
+User-space get absolute time: 1573486491.688454
+Kernel-space get absolute time: 1573486491.688420
+
+$ ./myprog 
+User-space get absolute time: 1573486494.69452
+Kernel-space get absolute time: 1573486494.69421
```

Comment:
is it 1573486494 sec  69421 microseconds or
1573486494 sec 694210 microseconds?